contracts/apps/ownership/OwnershipApp.sol

```diff
@@ -1,59 +1,87 @@
 pragma solidity ^0.4.11;
 
 import "../Application.sol";
-import "../../kernel/organs/TokensOrgan.sol";
 import "../../kernel/organs/ActionsOrgan.sol";
 import "../../misc/Requestor.sol";
 import "../../tokens/MiniMeIrrevocableVestedToken.sol";
 
 import "zeppelin/token/ERC20.sol";
 
-// OwnershipApp requires TokensOrgan and ActionsOrgan to be installed in DAO
+// OwnershipApp requires ActionsOrgan to be installed in DAO
 
 // At the moment OwnershipApp intercepts MiniMe hook events, if governance app
 // needs them, it has to have higher priority than ownership app
 
-contract OwnershipApp is Application, Controller, Requestor {
+contract OwnershipConstants {
+  bytes4 constant addTokenSig = bytes4(sha3('addToken(address,uint256,uint128,uint128)'));
```

Comment:
Can we make constants all caps?
`bytes4 constant ADD_TOKEN_SIG = bytes4(sha3('addToken(address,uint256,uint128,uint128)'));`
http://solidity.readthedocs.io/en/develop/style-guide.html#constants